### PR TITLE
Updates/fixes by Zyspec

### PR DESCRIPTION
**Pull request by mambax7**

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+    'renameTables' => [//         'XX_archive'     => 'ZZZZ_archive',
+    ],
+    'moduleStats'  => [
+        //            'totalcategories' => $helper->getHandler('Category')->getCategoriesCount(-1),
+        //            'totalitems'      => $helper->getHandler('Item')->getItemsCount(),
+        //            'totalsubmitted'  => $helper->getHandler('Item')->getItemsCount(-1, [Constants::PUBLISHER_STATUS_SUBMITTED]),
+    ],
+    'modCopyright' => "<a href='https://xoops.org' title='XOOPS Project' target='_blank'>
+                     <img src='" . constant($moduleDirNameUpper . '_AUTHOR_LOGOIMG') . "' alt='XOOPS Project'></a>",
```
Comment (geekwright) [inline config/config.php]: Updating the module results in

Warning: constant(): Couldn't find constant TAG_AUTHOR_LOGOIMG in file /modules/tag/config/config.php line 86

Comment (mambax7) [reply config/config.php]: OK, I'm on it....

File: view.tag.php [hunk 2]
```diff
@@ -12,163 +12,165 @@
...
 
 require_once __DIR__ . '/header.php';
 
-//xoops_loadLanguage('main', 'tag');
-/*
-if (!($GLOBALS["xoopsModule"] instanceof XoopsModule) || "tag" != $GLOBALS["xoopsModule"]->getVar("dirname", "n")) {
-    xoops_loadLanguage("main", "tag");
+//@todo refactor this code - it "works" but it's not right. Look at previous revs using $args_num to see what it's suppose to do
+if (Utility::tag_parse_args($args, $args_string)) {
+    $args['tag']  = !empty($args['tag']) ? $args['tag'] : (!empty($args_string[0]) ? $args_string[0] : Constants::DEFAULT_ID);
```
Comment (geekwright) [inline view.tag.php]: In use, looking at a tag "bbb":
Tag links are built as *modules/tag/view.tag.php?bbb*
but, following link results in a message: "Invalid query"

The "Jump to" box works and will get to "bbb" tagged items

It seems to work by changing line 30 in view.tag.php to:
```
    $args['tag']  = !empty($args['tag']) ? $args['tag'] : (is_numeric($args_string[0]) ? $args_string[0] : Constants::DEFAULT_ID);
//                                                         ^^^^^^^^^^
```
Without the change both $args['tag'] and $args['term'] get set to "bbb" and it results in an "Invalid query" message.

Comment (mambax7) [reply view.tag.php]: Thanks! I've included your fix

**Pull request merged**